### PR TITLE
Remove temporary CMake solution and use updated tbbbind_2_4 package

### PR DESCRIPTION
**Pull request by ivankochin**
Using updated TBBBind_2_4 package and remove a temporary solution that was applied in #5535.

### CODE REVIEW

File: inference-engine/cmake/dependencies.cmake [hunk 1]
```diff
@@ -133,6 +133,18 @@ if (THREADING STREQUAL "OMP")
...
+if (NOT ANDROID AND NOT(LINUX AND AARCH64))
+    if(DEFINED ENV{THIRDPARTY_SERVER_PATH})
+        set(IE_PATH_TO_DEPS "$ENV{THIRDPARTY_SERVER_PATH}")
+    elseif(DEFINED THIRDPARTY_SERVER_PATH)
+        set(IE_PATH_TO_DEPS "${THIRDPARTY_SERVER_PATH}")
+    else()
+        message(WARNING "THIRDPARTY_SERVER_PATH is not set (env or cmake cmd-line define). Pre-production TBB (with Hybrid support) cannot be loaded!")
+    endif()
+    message(STATUS "THIRDPARTY_SERVER_PATH=${IE_PATH_TO_DEPS}")
+endif()
```
Comment (ilya-lavrenov) [inline inference-engine/cmake/dependencies.cmake:146]: setting this globally, overrides GNA default downloading paths

Comment (ilya-lavrenov) [reply inference-engine/cmake/dependencies.cmake:146]: looks like the proper way is to upload to download01.org

CC @asuhov

Comment (ivankochin) [reply inference-engine/cmake/dependencies.cmake:146]: @asuhov, @ilya-lavrenov isn't a proper way is just to add the GNA package to the testing server? Can someone help me with it?

Comment (ivankochin) [reply inference-engine/cmake/dependencies.cmake]: The package was updated.

**Comment by ivankochin** [comment]
The CI was successfully passed on the testing server, the testing server paths were removed from the patch. The patch may be merged after updating packages on the production server.

**Review by myshevts — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**